src/rest_settings.ts

```diff
@@ -142,6 +142,108 @@ const paths = {
    * @method get
    */
   getCorrelationResults: `/correlations/${params.correlationId}/results`,
+  /*
+   * Waits for the ProcessStarted notification and returns the result
+   * @tag Notification
+   * @method get
+   */
+  onProcessStarted: '/notifications/long_polling/processStarted',
```

Comment:
Bitte entweder snake case, oder camel case, aber nicht gemischt.

Entweder: 

```suggestion
  onProcessStarted: '/notifications/long_polling/process_started',
```

oder:

```suggestion
  onProcessStarted: '/notifications/longPolling/processStarted',
```